bin/install_requirements_linux

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+RUBY_VERSION='2.3.1'
+NODE_VERSION='6.2.1'
+
+RED='\033[0;31m'
+PURPLE='\033[0;35m'
+RESET='\033[0m'
+script="$(basename $0)"
+
+log() {
+  echo -e "${PURPLE}${script}:${RESET} $1"
+}
+
+fail() {
+  log -e "${RED}${script} failed:${RESET} $1" >&2
+  exit 1
+}
+
+cd "$( dirname "$0" )/../requirements"
+requirements_dir="$( pwd )"
+
+log "\n\n====> Linux Development Setup <===="
+log "\nEach group of packages will be install one at a time. Follow the prompts to continue.\n"
+read -p "Press enter to continue"
```

Comment:
I think this stopping periodically is good practice, just to make sure students understand what's going on. Let's port these over to the other install files.